Lib/test/test_poll.py

```diff
@@ -130,7 +130,7 @@ def test_poll2(self):
         p = proc.stdout
         pollster = select.poll()
         pollster.register( p, select.POLLIN )
-        for tout in (0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
+        for tout in (0, 500, 1000, 2000, 4000, 8000, 16000, None, -1000) + (-1,)*3:
```

Comment:
I think that the total number of infinity timeouts should be not less than 10.  There are no guaranties that all finite timeouts are not expired before the subprocess will write the first line.

Reply:
@serhiy-storchaka Oh, sorry, now I got what you meant, updating the patch.